Expression_Tree/Expression_Tree.cpp

```diff
@@ -0,0 +1,112 @@
+#include<bits/stdc++.h>
```

Comment:
Please add a brief description of the expression tree.

Reply:
After doing the changes please squash the commits.

Reply:
I think a brief description should be in `README.md`.

Reply:
Yes, but 1-2 lines about what you are doing in the code will be nice.

Reply:
Done @somya-kapoor